rss2email/config.py

```diff
@@ -48,6 +64,18 @@ def __init__(self, dict_type=_collections.OrderedDict,
         super(Config, self).__init__(
             dict_type=dict_type, interpolation=interpolation, **kwargs)
 
+    def __getitem__(self, section_name):
+        """Show a section with overrides enabled. This is usually what is
+        wanted, since the user can configure the program with
+        overrides on the command line (via -s/--set).
+        """
+        return SectionOverrideProxy(section_name, self)
+
+    def get_real_section(self, section_name):
+        """This function shows the section as it appears on disk.
+        """
+        return super().__getitem__(section_name)
```

Comment:
Exposing this is a breach of abstraction really; if you go through all the pain of setting up a proxy class, the class should contain all logic related to merging of configs and stuff.

Reply:
Yes, I think it is in fact the `Config` object that should manage the merging of configs when written since all writes go through the `Config.write()` method anyway, so we can just override that.